modules/simulation/JsDCSupplySimulator/index.js

```diff
@@ -156,8 +153,9 @@ boot_module(async ({
 
     if (connector_current > 0) energy_import_total += (connector_voltage * connector_current * 0.5) / 3600;
     if (connector_current < 0) energy_export_total += (connector_voltage * -connector_current * 0.5) / 3600;
-    
+
     mod.provides.powermeter.publish.powermeter(power_meter_external());
+    mod.provides.main.publish.capabilities(Capabilities);
```

Comment:
Here, the capabilities are published every 500ms? They do not change in the SIL. That's why I would call  `publish.capabilities(Capabilities);` before the `setInterval()`.

Reply:
ok

Reply:
done